pkg/oci/client/oci.go

```diff
@@ -214,13 +269,36 @@ func (c *client) findInstanceByNodeNameIsDisplayName(nodeName string) (*baremeta
 	return &running[0], nil
 }
 
+// GetDriverDirectory gets the ath for the flexvolume driver either from the
+// env or default.
+func getCacheDirectory() string {
+	path := os.Getenv("OCI_FLEXD_CACHE_DIRECTORY")
+	if path != "" {
+		return path
+	}
+
+	path = os.Getenv("OCI_FLEXD_DRIVER_DIRECTORY")
+	if path != "" {
+		return path
+	}
+
+	return "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/oracle~oci"
+}
+
 // GetInstanceByNodeName retrieves the baremetal.Instance corresponding or a
```

Comment:
Should be "GetInstanceByNodeName retrieves the corresponding baremetal.Instance"